front/tests/assets/operationStudies/scenario.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": 0,
+  "name": "string",
+  "description": "string",
+  "tags": [
+    "string"
+  ],
+  "infra": 0,
+  "creation_date": "2023-02-02T23:31:08.218Z",
+  "last_modification": "2023-02-02T23:31:08.218Z",
+  "timetable": 0,
+  "trains_count": 0,
+  "trains_schedules": [
+    "string"
+  ]
+}
```

Comment:
missing new line

Reply:
Corrected all end lines in json (activated option in vscode)